.github/workflows/continuous-integration.yml

```diff
@@ -79,6 +79,12 @@ jobs:
           - "8.1"
           - "8.2"
           - "8.3"
+        include:
+          # Keep the locked version by default
+          - dependency-versions: "locked"
+          # For PHP 8.0, installing with --prefer-highest to use brick/math v0.11
+          - php-version: "8.0"
+            dependency-versions: "highest"
```

Comment:
Since brick/math is now locked at v0.12.1 (which requires PHP >= 8.1) in composer.lock, CI can not install it on PHP 8.0. Using --prefer-highest to avoid the version and use v0.11. Let me know if there is much better solution.

Reply:
maybe drop support for PHP 8.0

https://www.php.net/supported-versions.php

Reply:
I agree, PHP 8.0 is now EOL so we can remove the support.
Let @ramsey decide anyway.